src/rt/msvc_math.c

```diff
@@ -0,0 +1,52 @@
+/**
+* This module provides alternate implementations of single-precision math
+* functions missing in at least some 32-bit x86 MS VC runtimes
+*
+* Copyright: Copyright Digital Mars 2015.
+* License: Distributed under the
+*      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+*    (See accompanying file LICENSE)
+* Source:    $(DRUNTIMESRC rt/_msvc_math.c)
```

Comment:
I think the module names would be more consistent with `sections_*.d` and `deh_*.d` if this module is called `math_msvc.d`.